tools/virt-host-validate-ch.c

```diff
@@ -0,0 +1,129 @@
+#include <config.h>
+
+#include "virt-host-validate-ch.h"
+#include "virt-host-validate-common.h"
+#include "virarch.h"
+#include "virbitmap.h"
+
+int validateMinimumChVersion(char* ver_string)
+{
+	// expected ver_string example: "cloud-hypversor v0.8.1-<hash>"
+    int min_major_version = 0;
+    int min_minor_version = 9;
+    char *ptr = strtok(ver_string, " ");
+    char *last_tok = NULL;
+    while(ptr != NULL)
+	{
+        last_tok = ptr;
+		ptr = strtok(NULL, " ");
+	}
+    if (last_tok == NULL)
+        return -1;
+
+    // eliminate 'v'
+	last_tok++;
+    ptr = strtok(last_tok, ".");
+	if (ptr == NULL)
+		return -1;
+    int majorVer = atoi(ptr); 
+    ptr = strtok(NULL, ".");
+	if (ptr == NULL)
+		return -1;
+	int minorVer = atoi(ptr);
+
+	if (majorVer > min_major_version)
+	{
+		return 0;
+	}
+	else if (majorVer == min_major_version)
+	{
+		if (minorVer >= min_minor_version)
+			return 0;
+		else
+			return -1;
+	}
+	else
+	{
+		return -1;
+	}
+}
```

Comment:
Is there a reason this is being done differently than chExtractVersionInto? I did get a little feedback for simplifying that even in the RFC review [here](https://www.redhat.com/archives/libvir-list/2020-September/msg00887.html).

Reply:
After reviewing it again, I'd decide to remove this check as it is already covered by driver already. Instead, this place should only do environment check.